bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo_fi.properties

```diff
@@ -459,7 +459,7 @@ config.refreshInterval.description = Netatmo API\:n kyselyn päivitysväli (seku
 
 conf-error-no-client-id = Ei voida yhdistää Netatmo-siltaan, koska asiakastunnusta (client id) ei ole määritelty
 conf-error-no-client-secret = Ei voida yhdistää Netatmo-siltaan, koska salaustunnusta (secret) ei ole määritelty
-conf-error-grant-needed = Viimeistele määrittely myöntämällä lisäosalle pääsy Netatmo Connectiin\: ''{0}''.
+conf-error-grant-needed = Viimeistele määrittely myöntämällä lisäosalle pääsy Netatmo Connectiin\: {0}
```

Comment:
Please revert this change, it will have to be done at Crowdin, see https://v40.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
I have removed this from the commit